Makefile

```diff
@@ -160,6 +165,13 @@ BOOT_PART=$(INSTALLER)/boot
 BSP_IMX_PART=$(INSTALLER)/bsp-imx
 
 SBOM?=$(ROOTFS).spdx.json
+
+ROOTFS_YML=images/rootfs.yml.in
```

Comment:
You don't need to set twice if you use else:
```make
ifeq ($(HV),kubevirt)
ROOTFS_YML=images/kubevirt-rootfs.yml.in
else
ROOTFS_TML=images/rootfs.yml.in
endif
```

Reply:
Yeah that's a programming habit. I can fix that if that concerns you.